### PR TITLE
Feature/basic UI

### PR DESCRIPTION
**Pull request by bienvenuushindi**
- [ ] Create an app.rb file that will serve as your console app entry-point. It should have methods that do the following:
    - List all books.
    - List all people.
    - Create a person (teacher or student, not a plain Person).
   -  Create a book.
   - Create a rental.
   - List all rentals for a given person's id.

- [ ] In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
    - Present the user with a list of options to perform.
    - Lets users choose an option.
    - If needed, ask for parameters for the option.
    - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    list.length.positive? ? list : 'No Element Found'
+  end
+
+  def list_rentals(id)
+    puts 'Rentals: '
+    @rentals.each do |item|
+      puts "Date #{item.date}, Book \"#{item.book.title}\" by #{item.person.name}" if item.person.object_id == id
+    end
+  end
```
Comment (fobadara) [inline app.rb:35]: [REQUIRED] The list of rentals is not displayed after I enter the person's ID

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @bienvenuushindi, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
[OPTIONAL] Good job following some of the best practices for commits. But I think for a as large as this you should have more commits.

[OPTIONAL]  Good job adding the necessary headers in README but you didn't tell us how to use your app. You can have a look at [this repo](https://github.com/fobadara/quest-student-library/blob/development/README.md) for some ideas.

[OPTIONAL] You don't need to include `feature/...` in your branch name

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (bienvenuushindi) [reply app.rb:35]: @fobadara Thank you for your comment,
Kindly recheck my app  I think you probably entered the wrong person's ID [either non-existing or not linked to any rental], and that is why you can't see any result.
Below  is a screenshot that demonstrates my list of rentals functionality
showcase

As you can see from this screenshot you only get results when you enter a person's ID with rentals

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @bienvenuushindi 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Great communication with the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**